test/functional/generate-cert.prereq

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/env bats
 
-load "swupdlib"
+load "testlib"
 
+test_setup() {
+	:
+}
+
+test_teardown() {
+	:
+}
 
 @test "Generate certificate for signing Manifest.MoM" {
-  run sudo sh -c "rm -rf $CERT $BATS_TEST_DIRNAME/private.pem"
-  generate_cert
+
+  run sudo sh -c "rm -rf "$FUNC_DIR"/private.pem "$FUNC_DIR"/Swupd_Root.pem"
+  generate_certificate "$FUNC_DIR"/private.pem "$FUNC_DIR"/Swupd_Root.pem
```

Comment:
Functionally, there are some differences between 'generate_cert' and 'generate_certificate'.

1. Subjects are different: Does this matter for the MoM certificate? I had to set CN=localhost to get a webserver to work with the generated certificate.

2. The 'generate_certificate' function doesn't consume the certattributes.cnf file. Maybe this is necessary for the MoM certificate? If we don't consume this file with 'generate_certificate', then it is no longer used.

Reply:
you are right, I missed the part where the generate_cert function was using the certattributes.cnf file. To be honest I have no idea if this is needed or not, I don't know much about certificates, but I'll add as a optional parameter to the generate_certificate function. Good catch @jwakre !